common.mk

```diff
@@ -112,6 +112,7 @@ $(FIRRTL_FILE) $(ANNO_FILE) $(CHISEL_LOG_FILE) &: $(SCALA_SOURCES) $(SCALA_BUILD
 		--name $(long_name) \
 		--top-module $(MODEL_PACKAGE).$(MODEL) \
 		--legacy-configs $(CONFIG_PACKAGE):$(CONFIG) \
+		$(UPF_ASPECT) \
```

Comment:
I think this should be called `ASPECT_ARGS` since this can arbitrarily be any number of aspects.